openapi.yaml

```diff
@@ -83,108 +121,85 @@ parameters:
-        description: Page number of the returned results
-        example: 1
-      currentPageSize:
-        type: integer
-        description: Number of results per page
-        example: 25
   SelfLink:
     properties:
       self:
         type: string
         format: url
         description: Self-link of current resource
-  PaginationLinks:
-    properties:
-      first:
-        type: string
-        format: url
-        description: The first page of data
-      last:
-        type: string
-        format: url
-        description: The last page of data
-      prev:
-        type: string
-        format: url
-        description: The previous page of data
-      next:
-        type: string
-        format: url
-        description: The next page of data
-  PetResource:
+  TermResource:
     properties:
       id:
         type: string
-        description: A unique ID of pet
+        description: A 6-digit unique term code
       type:
         type: string
-        enum: [pet]
+        enum: [term]
       links:
         $ref: '#/definitions/SelfLink'
       attributes:
         properties:
-          name:
+          description:
+            type: string
+            description: Human readable academic term description
+            example: Winter 2019
+          season:
+            type: string
+            description: Season of the term
+            example: Winter
+            enum: ['Fall', 'Winter', 'Summer', 'Spring']
+          calendarYear:
+            type: string
+            description: A 4-digit calendar year
+            example: 2019
+          academicYear:
             type: string
-            description: Name of pet
-            example: Hedwig
-          owner:
+            description: Academic year associated with this term. This is a 4-digit code
+            example: 1819
+          financialAidYear:
             type: string
-            description: Name of owner
-            example: Harry Potter
-          species:
+            description: Aid year associated with this term. This is a 4-digit code. For example, for Summer 2018, the value would be 1819
+            example: 1819
+          startDate:
             type: string
-            description: Species of pet
-            example: Owl
-  PetResult:
+            description: First day of classes in this term. Format is in YYYY-MM-DD
+            format: date
+            example: 2019-01-07
+          endDate:
+            type: string
+            description: Last day of classes in this term. Format is in YYYY-MM-DD
+            format: date
+            example: 2019-03-22
+          housingStartDate:
+            type: string
+            description: First day for meal plan and housing assignments. Format is in YYYY-MM-DD
+            example: 2019-01-06
+          housingEndDate:
+            type: string
+            description: Last day for meal plan and housing assignments. Format is in YYYY-MM-DD
+            example: 2019-03-16
```

Comment:
Could you add `format: date` to `housingStartDate` and `housingEndDate` since they follow this format?